README.md

```diff
@@ -224,6 +225,14 @@ Specify that the git tag `2.4.2` of `stdlib' should be checked out:
       symlinks:
         my_module: "#{source_dir}"
 
+Move content of `manifests` directory to the root:
+
+    fixtures:
+      repositories:
+        stdlib:
+          repo: "git://github.com/puppetlabs/puppetlabs-stdlib"
```

Comment:
I would avoid using an example that does not have a crazy structure. Just use a fictitious example I think.